gossipsub_test.go

```diff
@@ -704,7 +726,7 @@ func TestGossipsubGraftPruneRetry(t *testing.T) {
 }
 
 func TestGossipsubControlPiggyback(t *testing.T) {
-	t.Skip("travis regularly fails on this test")
+	//t.Skip("travis regularly fails on this test")
```

Comment:
@aschmahmann I'm not being able to understand the last line of this test & of `TestGossipsubGossip`

```
// and wait for some gossip flushing
	time.Sleep(time.Second * 2)
```

Please can you explain what this is for ? Why is it important to flush the remaining gossip/control messages once the test is complete ?

Reply:
Not really sure, perhaps it's just supposed. Hopefully either @vyzo can shed some light or perhaps there's an answer to be found in the first gossipsub PR #67.